samples/subsys/fs/littlefs/src/main.c

```diff
@@ -68,6 +68,9 @@ void main(void)
 
 	flash_area_close(pfa);
 
+	/* Do not mount if auto-mount has been enabled */
+#if !DT_NODE_EXISTS(PARTITION_NODE) ||						\
+	!(FSTAB_ENTRY_DT_MOUNT_FLAGS(PARTITION_NODE) & FS_MOUNT_FLAG_AUTOMOUNT)
```

Comment:
It might make sense to have `fs_mount()` return `-EALREADY` in this case.

Otherwise I have no comment other than there's at least some sample somewhere where this worked.

Reply:
We should probably alter `fs_mount` and backends to return `-EALREADY`, right now I think it returns `-EBUSY`, which is also used when for example littlefs structures given to mount are already in use by other instance.